docs/distributing-policies.md

```diff
@@ -83,28 +90,30 @@ $ kwctl annotate policy.wasm \
     --output-path annotated-policy.wasm
 ```
 
-This process performs some optimizations on the policy, so it's not
-uncommon to end up with a smaller annotated policy than the original
-one. This depends a lot on the toolchain that was used to produce the
-unannotated WebAssembly object.
+This process performs some optimizations on the policy, so often
+the annotated policy is smaller than the original.
+It depends considerably on the toolchain that was used to produce the
+original WebAssembly object.
 
-You can check with `kwctl inspect` that everything looks correct:
+You can check with `kwctl inspect` that everything is correct:
 
+<!--TODO: An actual example here? What to look for when correct/incorrect?-->
 ```shell
 $ kwctl inspect annotated-policy.wasm
 # here you will see a colored output of the metadata you provided on the `metadata.yml` file. This information is now read from the WebAssembly custom sections
 ```
 
 ## Pushing the policy
 
-Pushing an annotated policy can be done in this way:
+Pushing an annotated policy can be done like this:
 
 ```shell
 $ kwctl push annotated-policy.wasm \
               <oci-registry>/kubewarden-policies/palindromify-policy:v0.0.1
 ```
 
-It is discouraged to push unannotated policies. This is why by default
+<!--TODO: A why or a reference would be good for this next paragraph.-->
+It is discouraged to push unannotated policies. This is why, by default,
 `kwctl push` will reject to push such a policy to an OCI registry. If
 you really want to push an unannotated policy you can use the
 `--force` flag of `kwctl push`.
```

Comment:
One reason of why  unannotated policies are not allowed to be pushed by default is the fact the policy server uses this metatada to properly run te policies. For example, the metadata has the flag telling if the policy is a mutating policy or not.

Reply:
ok, thanks.